webclient/components/ptinfo-single/1time-Mrow-mField/diagnosis/db/full-sync-with-server-db-mixin.js

```diff
@@ -0,0 +1,12 @@
+// Reference implementation
+import clientSideTblPatientDiagnosis from './client-side/structure/patient-table-of-diagnosis'
```

Comment:
watch how reminders have been initialized. Also there is a youtube video that shows how the initialization should be done.